src/lib/cm/prompts/createCandyMachineWizardPrompt.ts

```diff
@@ -275,7 +283,7 @@ const createCandyMachinePrompt = async (): Promise<{ candyMachineConfig: CandyMa
             const groupGuards: RawGuardConfig = {}
 
             const selectedGuards: string[] = await checkbox({
-                message: `Select the guards to assign to group ${groupName} (press space to select, or select Quit to abort):`,
```

Comment:
Why did you remove "(press space to select, or select Quit to abort)" for both occurrences? I personally preferred to have it in for ease of use.